activesupport/lib/active_support/lazy_load_hooks.rb

```diff
@@ -68,7 +68,11 @@ def execute_hook(name, base, options, block)
           if options[:yield]
             block.call(base)
           else
-            base.instance_eval(&block)
+            if base.is_a?(Class) || base.is_a?(Module)
```

Comment:
Is a module a subclass of Class or is it the other way around? I’m pretty sure one of these could be skipped.

Reply:
Yes, a class is a module. I'll fix it.